user-service/src/main/java/rs/edu/raf/banka1/dtos/CurrentAccountDto.java

```diff
@@ -0,0 +1,21 @@
+package rs.edu.raf.banka1.dtos;
+
+import lombok.Getter;
+import lombok.Setter;
+
```

Comment:
If accepted, changes mentioned inside the model package should be applied to DTO classes to avoid any errors.